modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectScopeProvider.java

```diff
@@ -25,6 +25,8 @@ public long getGroupId(HttpServletRequest httpServletRequest)
 
 	public String[] getRootPanelCategoryKeys();
 
+	public Long getValidGroupId(long companyId, String scopeKey);
+
```

Comment:
@devpedrotavares did you run `gw baseline` after this change?

Reply:
I missed that, thanks. I just ran it now and added the breaking change commit message.